spec/requests/evaluators_spec.rb

```diff
@@ -41,17 +45,17 @@
       end
 
       it 'calls the EvaluatorManagementService to process the invitation' do
-        expect(evaluator_service).to receive(:process_evaluator_invitation).and_return({ success: true,
-                                                                                         message: 'Invitation sent successfully.' })
-        post phase_manage_evaluators_path(phase), params: valid_params
-        expect(response).to redirect_to(phase_manage_evaluators_path(phase))
+        expect(evaluator_service_double).to receive(:process_evaluator_invitation).and_return({ success: true,
```

Comment:
I wasn't sure about creating this test double for most of these specs but I left it in anyways. could be fine.

Reply:
Good question - yeah just separating concerns, by mocking the service here I wanted to ensure the request specs were able to isolate and test the controller's behavior and response handling, rather than only testing the business logic in the service - which then is tested in the `spec/services/evaluator_management_service_spec.rb` anyways.